### DISCUSSION TITLE
Multi App at the same time running:

### DISCUSSION THREAD

**Discussion by angelogl** [serenity-is/Serenity#7045] [General]
I've developed two apps in Serenity Framework, and I need to use both at the same time. But, when I log into one of them and try to use the other, it disconnects me.

**Comment by Surenthiran** [comment]
two apps in same solution? or two apps hosted in same place? we need more information about issue!

**Reply by angelogl** [reply]
Yes, two apps hosted in the same server. I think it has something with the cookies management.

**Reply by reach2rv** [reply]
Add  `"Urls":"http://*:port" to app settings`

Make sure you define unique port for each instance

**Reply by angelogl** [reply]
Thanks a lot. It works.
